main.js

```diff
@@ -43,6 +44,22 @@ function createMainWindow() {
     });
   }
 
+  // // Add CSP headers
```

Comment:
These broke devtools, or something else webpack-y.

Can't work out how to fix it. Google isn't helpful.

Reply:
Maybe add a listener in a seperate .js file to  open links in default browser? I have done this in the past, works fine :)

Reply:
Sorry, I meant the CSP headers -- the external link opening is fine @GewoonJaap